CGPs/CGP-0043.md

```diff
@@ -0,0 +1,60 @@
+---
+cgp: 0043P
+title: Reserve freezing to get back to target allocation
+date-created: 2021-10-12
+author: @martinvol, @rcroessmann
+status: DRAFT
+governance-proposal-id: [if submitted]
+date-executed: [if executed] <date created on, in ISO 8601 (yyyy-mm-dd) format>
+---
+## Overview
+
+CGP 0043P - Reserve freezing to get back to target allocation
+
+This proposal is meant to freeze a portion of the unfrozen Celo allocation the reserve currently holds, proposing a multi-year linear release schedule.
+
+As Mento bucket sizes are proportional to the Reserve size, in order to keep the current bucket sizes as-they-are, it is needed to increase the reserveFraction for all the stable assets currently deployed (cUSD and cEUR) in the same proportion as the percentage of the Reserve that is frozen.
+
+### Status
+
+According to the [Reserve Website](https://celoreserve.org/), it currently has a target allocation as follows:
+- 50% CELO
+- 29.5% BTC
+- 15% ETH
+- 5% Stable Value Assets*
+- 0.5% Natural Capital Backed Assets*
+
+However, the current allocation the Reserve, counting only unfrozen allocation, has of currently ~68% allocation of CELO. This was caused by an original linear schedule of two years (set in [this proposal](https://github.com/celo-org/governance/blob/main/CGPs/cgp-0006.md)) being too aggresive in practice.
```

Comment:
Add to: the current allocation `to` the Reserve
`has of currently` to `currently has`

Reply:
```suggestion
However, the current allocation the Reserve, counting only unfrozen allocation, currently has a ~68% allocation of CELO. This was caused by an original linear schedule of two years (set in [this proposal](https://github.com/celo-org/governance/blob/main/CGPs/cgp-0006.md)) being too aggresive in practice.
```